### PR TITLE
For #23574: Remove stale partial bitmaps created while downloading wallpapers

### PR DESCRIPTION
**Pull request by Amejia481**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: .experimenter.json [hunk 1]
```diff
@@ -18,14 +18,25 @@
       "message-location": {
         "description": "Where is the message to be put.",
         "enum": [
+          "app-menu-item",
```
Comment (MozillaNoah) [inline .experimenter.json:21]: accidental commit?

**Review by MozillaNoah — APPROVED** [review]
LGTM

File: .experimenter.json (hunk 1 above)
Comment (MatthewTighe) [reply .experimenter.json:21]: this `.experimenter.json` stuff is killing me too, lol

**Review by MatthewTighe — APPROVED** [review]

File: .experimenter.json (hunk 1 above)
Comment (Amejia481) [reply .experimenter.json:21]: Sorry about that XD

Comment (gabrielluong) [reply .experimenter.json:21]: Doesn't matter if we commit this or not. The ordering is non-deterministic so it will keep flipping regardless. 😂

**Pull request merged**